.github/pull_request_template.md

```diff
@@ -34,7 +34,4 @@ Please delete options that are not relevant.
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes
```

Comment:
I think it isn't necessary as we got the github pipeline, which should pass strictly to be able to merge ready.